addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -74,9 +74,14 @@ function factory(dependencies) {
             this._onKeyUp = this._onKeyUp.bind(this);
             browser.addEventListener('keydown', this._onKeyDown);
             browser.addEventListener('keyup', this._onKeyUp);
-            browser.addEventListener('beforeunload', async () => {
-                this.channel && await this.channel.leaveCall();
-            });
+            /**
+             * Disconnects the RTC session if the page is closed or reloaded,
+             * it is done in both 'unload' and 'beforeunload' as browsers may
+             * trigger only one of these events in different cases.
+             */
+            this._onAnyUnload = this._onAnyUnload.bind(this);
+            browser.addEventListener('beforeunload', this._onAnyUnload);
+            browser.addEventListener('unload', this._onAnyUnload);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Window/unload_event

This does not look very recommended, and as it doesn't really fix the issue for me, I'm not sure if we really need this?

Reply:
I think we can close this PR and I'll try with AL's suggestion for a date filter.

Reply:
I still like the rest of the clean up in this PR, just not adding this event then.